src/store/configureStore.js

```diff
@@ -0,0 +1,23 @@
+import { compose, createStore, applyMiddleware } from 'redux';
+import { routerMiddleware } from 'react-router-redux';
+import createSagaMiddleware, { END } from 'redux-saga';
+import rootReducer from '../Root/rootReducer';
+
+function configureStore({ initialState = {}, history }) {
+  const route = routerMiddleware(history);
+  const sagaMiddleware = createSagaMiddleware();
+  const store = createStore(
+    rootReducer,
+    initialState,
+    compose(
+      applyMiddleware(route, sagaMiddleware)
+    ),
+  );
+
+  store.runSaga = sagaMiddleware.run;
+  store.close = () => store.dispatch(END);
+
+  return store;
+}
+
+export default configureStore;
```

Comment:
I have a question here. Why don't we `` import rootSaga from './Root/rootSaga'; `` and `` store.runSaga(rootSaga); `` in this file? Then we don't have to expose runSaga as a property to the store. What do you think?